client/asset/btc/rpcclient.go

```diff
@@ -770,6 +770,24 @@ func (wc *rpcClient) walletLock() error {
 // sendToAddress sends the amount to the address. feeRate is in units of
 // sats/byte.
 func (wc *rpcClient) sendToAddress(address string, value, feeRate uint64, subtract bool) (*chainhash.Hash, error) {
+	// 1e-5 = 1e-8 for satoshis * 1000 for kB.
+	var txid string
+	// if legacy sign, use legacy sendToAddress.
+	if wc.rpcCore.legacySignTx {
```

Comment:
This is an interesting inference.  Can it be justified with release notes that show the introduction of the new sign method coincides with the addition of the fee_rate arg for sendtoaddress?
Even if it can, why don't wee make a new `BTCCloneCFG` field for this?  There's a lot of RPC variation with zcashd, dogecoind, litecoind, etc. and we are generally checking each one explicitly.

Reply:
Ok sounds fine. How should it be named? `legacyForks`?

Reply:
I mean more specific, not general.  See the existing bools in `BTCCloneCFG`.  An appropriate name would be `LegacySentRawTx